cfme/fixtures/pytest_selenium.py

```diff
@@ -184,9 +184,24 @@ def element(o, **kwargs):
     Raises:
         NoSuchElementException: When element is not found on page
     """
+    no_deeper = kwargs.pop("_no_deeper", False)
```

Comment:
I think it's fine leaving this out of the docstring, but a comment explaining why this might be needed would be good.